setup.py

```diff
@@ -171,8 +171,7 @@ def _filter_requirements(lines_iter, filter_names=None,
 
 REQ_UPPER_BOUNDS = {
     'bcolz': '<1',
-    'pandas': '<0.19',
-    'pandas-datareader': '<0.6',  # 0.6.0 requires pandas >=0.19.2
+    'pandas': '<=0.22',
```

Comment:
Shouldn't this be "~=0.22" (or hopefully "~=0.23" if that works as well)? Otherwise bugfix releases of pandas will not be accepted.

Reply:
Without further changes, 0.23 didn't work, so I didn't add it to this PR (would love not to add more scope to this instead of merging what we have). 

We could make it `<0.23`, if we think it's safe to support the micro versions. We have seen in the past that micro versions of pandas have changed behavior and broken things...